documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -60,8 +60,14 @@ spec:
 |`10`
 
 |`controller_filesystem_overhead`
-|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this label can only be changed using the CLI
+|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this label can only be changed using the CLI.
```

Comment:
while on it, can you please replace `label` with `setting` here as well? (label has a different meaning in k8s/openshift)